Sprint-3/3-stretch/creditCard-validator.test.js

```diff
@@ -0,0 +1,29 @@
+const creditCardValidator = require("./creditCard-validator");
+
+describe("creditCardValidator", () => {
+  test("should return true for a 16 digit long card number", () => {
+    expect(creditCardValidator("1234-5678-9012-3456")).toBe(true);
+    expect(creditCardValidator("1234 5678 9012 3456")).toBe(true);
+  });
+
+  test("should return false if the card number is not digit only", () => {
+    expect(creditCardValidator("1234-5678-9012-345a")).toBe(false);
```

Comment:
Your only example here of an invalid number has a letter only in the last character. But you have _other_ checks that only check the last character as well. It's possible this is only returning false because you're just checking that the last character is 0/2/4/6/8. Can you think how you could add more tests to make sure this isn't the case?